components/switch/switch-mixin.js

```diff
@@ -0,0 +1,181 @@
+import '../colors/colors.js';
+import { css, html } from 'lit-element/lit-element.js';
+import { getUniqueId } from '../../helpers/uniqueId.js';
+import { ifDefined } from 'lit-html/directives/if-defined.js';
+import { RtlMixin } from '../../mixins/rtl-mixin.js';
+
+export const SwitchMixin = superclass => class extends RtlMixin(superclass) {
+
+	static get properties() {
+		return {
+			disabled: { type: Boolean, reflect: true },
+			label: { type: String, reflect: true },
```

Comment:
With the label (and labelHidden) being defined in the mixin, it means that it can be overridden even for the visibility switch. I'm fine with that, but just wanted to make sure that's what you intended.

Reply:
Thanks yep... that's the intent so far.